frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segments_view.tsx

```diff
@@ -1098,41 +1096,21 @@ class SegmentsView extends React.Component<Props, State> {
         Store.dispatch(updateIsosurfaceVisibilityAction(layerName, segment.id, isVisible));
       }
     });
-
-    const newSegmentsInGroupVisible = {
-      ...this.state.areSegmentsInGroupVisible,
-      [groupId]: isVisible,
-    };
-    this.setState({
-      areSegmentsInGroupVisible: newSegmentsInGroupVisible,
-    });
   };
 
   handleLoadMeshesAdHoc = (groupId: number) => {
     this.handlePerSegment(groupId, (segment) => {
       if (segment.somePosition == null) return;
       this.props.loadAdHocMesh(segment.id, segment.somePosition);
     });
-
-    const newSegmentsInGroupVisible = {
-      ...this.state.areSegmentsInGroupVisible,
-      [groupId]: true,
-    };
-    this.setState({
-      areSegmentsInGroupVisible: newSegmentsInGroupVisible,
-    });
   };
 
   getSegmentsWithMissingLocation = (groupId: number): number[] => {
```

Comment:
I'd replace `Location` with `Position` here to keep it consistent with the `somePosition` attribute.